organization/_posts/2019/2019-07-11-coordination.md

```diff
@@ -0,0 +1,313 @@
+        we have not discussed this yet in detail in any meeting and I
+        am not aware of any significant progress in this area. One WG
+        member recently informed us that he is starting some work on
+        this.
+
+### Event Delivery Forum
+  - Doodling for first meeting at the end of this month:
+      - [<span class="underline">https://doodle.com/poll/n3ykm6knp6tv5uif</span>](https://doodle.com/poll/n3ykm6knp6tv5uif).
+
+### Packaging
+  - Met yesterday, minutes available
+    [<span class="underline">here</span>](https://hepsoftwarefoundation.org/organization/2019/07/10/packaging.html).
+    We discussed the Key4HEP stack project.
+  - Next meeting 11 September.
+
+### Frameworks
+  - We have 9 nominees for the working group.
+  - Names have been circulated and we requested comments by 25 August.
+      - This is very long, but it’s summer and it would be hard to
+        converge and organise anything before this time.
+      - Comments to
+        [<span class="underline">hsf-wg-search-committee@cern.ch</span>](mailto:hsf-wg-search-committee@cern.ch)
+        (maps to Graeme, Liz and Michel)
+
+### Licensing
+  - 
+
+### Quantum Computing
+  - 
+
+### Visualisation
+  - Phoenix GSoC project is going very well.
+      - Github:
+        [<span class="underline">https://github.com/HSF/phoenix</span>](https://github.com/HSF/phoenix)
+      - Live demo:
+        [<span class="underline">http://hepsoftwarefoundation.org/phoenix/</span>](http://hepsoftwarefoundation.org/phoenix/)
+  - Experiment independent in browser toolkit, easy to use.
+  - Would be good to present this.
+
+### Software Forum
+  - 
+
+## Workshops
+
+### Pre-CHEP (2-3 November)
+  - Wait for next Analysis WG meeting for progress.
+
+### Next HSF/WLCG Workshop
+  - Call for hosting institutes is out.
+      - 3 interested so far.
+  - Considering the constraints in May and June
+      - June now looks very difficult (LHCC, LHCb Week, ATLAS S\&C
+        Week, ATLAS Week)
+      - In May 4-8 or 11-15 look possible (18-22 is Ascension Week,
+        25-29 DIRAC Users Workshop)
+  - Are there any more known clashes for **May 4-8** or **May 11-15**?
+  - It would be very good to pin down a favoured date *now*.
+      - We agreed that 11-15 May would be favoured.
+  - Can we extend the deadline for institutes?
+      - Early September should be ok - Graeme will discuss with the
+        other organisers.
+
+## AOB
+  - Git repository for Harvester.
+      - Harvester is PanDA’s new workload submission engine (deals
+        with HPCs, dynamic cloud resources, other non-standard
+        resources)
+      - But it’s actually backend agnostic, so a request to host the
+        code as an HSF project.
+          - Part of Intelligent Data Delivery System (iDDS).
+      - N.B. similar to things we currently do host, like CREST and
+        Phoenix.
+      - Agreed.
+  - Summer break now, when to resume meetings?
+      - Resume 5 September.
```

Comment:
Try and update the date if we converge. In any case it would either be Aug 29 or Sep 12.